win32ss/drivers/videoprt/resource.c

```diff
@@ -424,9 +424,12 @@ IntVideoPortMapMemory(
 
       return MappedAddress;
    }
-
-   if (Status)
-      *Status = NO_ERROR;
+   else
+   {
+      if (Status)
+         *Status = ERROR_NOT_ENOUGH_MEMORY;
```

Comment:
> IntVideoPortMapMemory incorrectly returns a successful result if it failed to map video memory.

It looks like there are other occurrences, like
https://github.com/reactos/reactos/blob/d1f54aa5f7277ed202e048253db71d319e2db497/win32ss/drivers/videoprt/resource.c#L381-L387